contracts/strategies/dnsVectorStrategy/DnsVectorStrategyAumLib.sol

```diff
@@ -160,4 +154,41 @@ library DnsVectorStrategyAumLib {
                 Math.SHORT_FIXED_DECIMAL_FACTOR) / 10**18) /
             (assetBalances[0].balance);
     }
+
+    function getPangolinLpBalance() public view returns (uint256) {
+        DnsVectorStorage storage strategyStorage = DnsVectorStorageLib
+            .getStorage();
+
+        return
+            strategyStorage
+                .pangolinMiniChef
+                .userInfo(strategyStorage.pangolinPoolId, address(this))
+                .amount;
+    }
+
+    function getPangolinLpReserve(address token0, address token1)
```

Comment:
Not a big fan of introducing dependencies between libraries in 'run-time': can we put this function into DnsVectorStrategyCommon and mark it as 'internal'? That way the dependency will be only 'compile time'.